apps/src/music/views/Timeline.tsx

```diff
@@ -111,9 +112,60 @@ const Timeline: React.FunctionComponent<TimelineProps> = ({
     return eventHeight > 8 ? 3 : eventHeight > 6 ? 2 : 1;
   }, []);
 
+  const timelineElements = Array.from(
+    document.querySelectorAll<HTMLElement>('#timeline-element')
```

Comment:
I'm a little wary of looking these up by a hardcoded element ID 🙂 could we create a constant for this? Another more involved option might be to pass functions to the ref attributes of the `TimelineElement`s: https://react.dev/learn/manipulating-the-dom-with-refs#how-to-manage-a-list-of-refs-using-a-ref-callback

Reply:
Awesome, thanks!